universe/auto_syncer.go

```diff
@@ -846,18 +846,30 @@ func (f *FederationEnvoy) SyncAssetInfo(ctx context.Context,
 		// Sync failures are expected from Universe servers that do not
 		// have a relevant universe root.
 		if err != nil {
-			log.Debugf("asset lookup for %v failed with remote"+
-				"server: %v", assetID.String(), addr.HostStr())
-			//lint:ignore nilerr failure is expected and logged
+			log.Warnf("Asset lookup for %v failed with remote"+
+				"server '%v': %v", assetID.String(),
```

Comment:
A space is missing between "remote" and "server" in the log message.

Reply:
nice catch

Reply:
Oops, thanks for noticing! Fixed.